src/plugins/tts/plugin.tts.js

```diff
@@ -158,18 +158,35 @@ BookReader.prototype.initNavbar = (function (super_) {
 
       $el.find('.BRcontrols').prepend(this.refs.$BRReadAloudToolbar);
 
+      const renderVoiceOption = (language) => {
+        return language.map(voice =>
+          `<option value="${voice.voiceURI}">${voice.lang} - ${voice.name}</option>`);
```

Comment:
We want this to return a plain string; using just `map` returns an array of strings!

```suggestion
          `<option value="${voice.voiceURI}">${voice.lang} - ${voice.name}</option>`)
          .join('');
```